com.github.KRTirtho.Spotube.yml

```diff
@@ -0,0 +1,46 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: com.github.KRTirtho.Spotube
+runtime: org.gnome.Platform
```

Comment:
asking the quesiton again: do you really need the GNOME runtime?

Reply:
My application uses the [`libwinmedia`](https://github.com/harmonoid/libwinmedia) plugin which depends on `libwebkit2gtk-4.0` & it doesn't exist in the `freedesktop` platform
Without GNOME runtime the audio_process of my application can't start